synapse/common.py

```diff
@@ -430,11 +430,12 @@ def tmpfile(dirn: typing.Optional[str] = None, prefix: typing.Optional[str] = No
     Args:
         dirn (Optional[str]): The optional directory name to create the tempfile in.
         prefix (Optional[str]): The optional tempfile name prefix.
+        mode (str): The mode to open the file with (default: 'wb').
```

Comment:
t```suggestion
        mode (str): The mode to open the file with.
```
Python API autodocs should take care of the default mode.